src/transforms/pull-request.ts

```diff
@@ -19,20 +19,21 @@ function mapReviews(reviews) {
 	// The reduce function goes through all the reviews and creates an array of unique users (so users' avatars won't be duplicated on the dev panel in Jira) and it considers 'APPROVED' as the main approval status for that user.
 	return sortedReviews.reduce((acc, review) => {
 		// Adds user to the usernames object if user is not yet added, then it adds that unique user to the accumulator.
-		if (!usernames[review.user.login]) {
-			usernames[review.user.login] = {
-				name: review.user.login,
+		const user = review?.user;
+		if (!usernames[user?.login]) {
```

Comment:
if user?.login is undefined wouldn't we get an error when trying to acccess a map by undefined key?

Reply:
I guess review must have user. According to the previous code. So I wonder why do we use `?.` here?

Reply:
you can do `someArray[undefined]`.  It'll returned `undefined` as the key doesn't exist.  I'm adding a lot of optional chaining because we don't have types to protect us and a lot of github docs are wrong as they're just missing parts of the payload depending of the situation.  I was just being cautious